src/Microsoft.AspNet.OData/Microsoft.AspNet.OData.csproj

```diff
@@ -42,6 +42,11 @@
       <HintPath>..\..\sln\packages\Microsoft.Spatial.7.12.2\lib\net45\Microsoft.Spatial.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.Spatial, Version=7.11.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
This line is still wrong. You can see Line 42, we have 7.12.2 referencing, but Line46 have 7.11.1 referencing again. 
I think you should rebase your PR to the latest master codes.

Same for newtonsoft.Json package reference.

Reply:
✅